P5/Exemplars/tei_jtei.odd

```diff
@@ -3262,26 +3271,26 @@
           </constraintSpec>
           <constraintSpec ident="jtei.sch-teiVersion" scheme="schematron">
             <constraint>
-              <sch:rule context="@target[matches(., '^https?://www\.tei-c\.org/release/doc/tei-p5-doc')]">
+              <sch:rule context="@target[matches(., '^https?://(www\.)?tei-c\.org/release/doc/tei-p5-doc')]">
```

Comment:
probably nitpicking, but you might want to escape the dashes in the regex as well?

Reply:
Ah, I wasn't even aware the hyphen could be escaped (except inside a character class), since it's interpreted as a literal in this context. Again, `-` seems more readable (and less confusing than a non-necessary character escape) here, unless you disagree.

Reply:
you're right, again!